app/services/register_records_downloader.rb

```diff
@@ -19,6 +19,14 @@ def download_format(format)
     end
   end
 
+  def download_ods
+    headers = register.fields_array
+    data = register.records.where(entry_type: 'user').find_each.map do |r|
+      headers.map { |f| r.data[f] }
+    end
+    SpreadsheetArchitect.to_ods(headers: headers, data: data)
```

Comment:
Is there anything else we want to provide in this spreadsheet? For example, a different sheet with metadata in it?

Reply:
Not at the moment. The goal of this format is to provide an alternative to CSV that can be open without ceremony in UTF-8.